api/client/cli.go

```diff
@@ -93,10 +93,11 @@ func (cli *DockerCli) Subcmd(name, signature, description string, exitOnError bo
 	flags := flag.NewFlagSet(name, errorHandling)
 	flags.Usage = func() {
 		options := ""
+		tempSignature := " " + signature
```

Comment:
Well, it depends on which spaces annoy you  :-)
You started out working on this because of the trailing spaces, right?
So as long as you got rid of those and didn't add double-spaces in the middle then I think its a step forward.  From my quick test I see these commands as adding extra spaces at the end:
Usage: docker events [OPTIONS] 
Usage: docker info 
Usage: docker load [OPTIONS] 
Usage: docker ps [OPTIONS] 
Usage: docker version 
if those are fixed then its good.